Resources/NetHookAnalyzer2/NetHookAnalyzer2/MainForm.cs

```diff
@@ -105,7 +106,7 @@ string GetLatestNethookDumpDirectory()
 			return Path.Combine(nethookDirectory, latestDump);
 		}
 
-		NetHookDump Dump { get; set; }
+		internal static NetHookDump Dump { get; private set; }
```

Comment:
`internal` I can live with, but I really don't like `static` here.

Reply:
What's the alternative? I don't want to pass the dump all over the place.